### PR TITLE
Part of 17670: Replace Typography with Text confirm-approve-content.component.js and home.component.js

### PR DESCRIPTION
**Pull request by spiritanand**
## Explanation
Replace Typography with Text `confirm-approve-content.component.js` and `home.component.js`

Part of: #17670
### Before
Before approve-content

### After
After approve-content

## Manual Testing Steps

- Go to the latest build of storybook on this PR
- Follow path `Pages/ConfirmApprove/ConfirmApproveContent` in the side panel
- Check `ConfirmApproveContent` story against develop storybook

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/pages/home/home.component.js [hunk 1]
```diff
@@ -300,20 +301,22 @@ export default class Home extends PureComponent {
...
                         >
                           {t('somethingWentWrong')}
-                        </Typography>
-                        <Typography
+                        </Text>
+                        <Text
                           color={TextColor.textAlternative}
-                          variant={TypographyVariant.H7}
+                          variant={TextVariant.bodySm}
+                          as="h7"
```
Comment (georgewrmarshall) [inline ui/pages/home/home.component.js:315]: There is no html element `h7` check mapping. Anything that is over `h6` should be `h6`
```suggestion
                          as="h6"
```

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Good start! Left some suggestions. Change all as="h7" to as="h6"

File: ui/pages/home/home.component.js [hunk 2]
```diff
@@ -1,68 +1,69 @@
...
+import React, { PureComponent } from 'react';
 import { Redirect, Route } from 'react-router-dom';
 ///: BEGIN:ONLY_INCLUDE_IN(main)
 import {
+  CONTEXT_PROPS,
   EVENT,
   EVENT_NAMES,
-  CONTEXT_PROPS,
 } from '../../../shared/constants/metametrics';
+import { SECOND } from '../../../shared/constants/time';
```
[the remark this answers is not in the document]
Comment (spiritanand) [reply ui/pages/home/home.component.js:10]: Sure.

File: ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js [hunk 3]
```diff
@@ -550,14 +546,15 @@ export default class ConfirmApproveContent extends Component {
...
-              boxProps={{ marginLeft: 1, marginTop: 2 }}
+              marginLeft={1}
+              marginTop={1}
             >
               {origin}
-            </Typography>
+            </Text>
           </Box>
         </Box>
         <div
```
[the remark this answers is not in the document]
Comment (spiritanand) [reply ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js:560]: what is the review for this line @georgewrmarshall

[the remark this answers is not in the document]
Comment (georgewrmarshall) [reply ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js:560]: I add screenshots of the before and after changes as a comment on the file so it's easy to see what file relates to which screenshot. Helps with review if you could do the same for the rest of the files that would be greatly appreciated!

[the remark this answers is not in the document]
Comment (spiritanand) [reply ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js:560]: ok sure. i will keep that in mind.

**Comment by spiritanand** [comment]
@georgewrmarshall are there any other changes that I ought to do? I am currently failing in one test, i.e. the e2e chrome build test.

**Review by georgewrmarshall — DISMISSED** [review]
LGTM 💯 Thanks for your contribution @spiritanand!
I believe the code coverage is not effected with these changes so happy to merge without any updates to the test coverage.

File: ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js [hunk 4]
```diff
@@ -543,21 +540,21 @@ export default class ConfirmApproveContent extends Component {
...
             />
-            <Typography
-              variant={TypographyVariant.H6}
-              fontWeight={FONT_WEIGHT.NORMAL}
+            <Text
+              variant={TextVariant.bodySm}
+              as="h6"
               color={TextColor.textAlternative}
-              boxProps={{ marginLeft: 1, marginTop: 2 }}
+              marginLeft={1}
```
Comment (georgewrmarshall) [inline ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js:554]: Hey @spiritanand, I just added a little alignment fix here so we can get this through! cc @garrettbear 

Screenshot 2023-03-10 at 10 10 24 AM
Screenshot 2023-03-10 at 10 09 51 AM

**Review by georgewrmarshall — APPROVED** [review]

**Review by garrettbear — APPROVED** [review]

**Pull request merged**